frontend/src/components/MonacoEditor/MonacoEditor.vue

```diff
@@ -115,7 +121,27 @@ const init = async () => {
         ?.getValueInRange(editorInstance.getSelection()) as string;
 
       const query = selectedValue || typedValue;
-      emit("run-query", query);
+      emit("run-query", { explain: false, query });
+    },
+  });
+
+  // add the run query action in context menu
+  editorInstance.addAction({
```

Comment:
@LiuJi-Jim I am wondering if we could unify the k-bar config into a single file like the route config so that it's easy to view shortcut

Reply:
@tianzhou Yes we can. In fact we are registering global k-bar actions in KBarWrapper.vue. But we are also registering and unregistering actions dynamically through different routes. Both static and dynamic actions are listed in k-bar.

Reply:
@tianzhou @LiuJi-Jim The shortcuts in KBar seem a bit hidden, If we have a shortcuts list, we can design a friendly tips UI to tell the users.